recipe/meta.yaml

```
@@ -29,7 +29,7 @@ requirements:
     - python
     - pyjulia >=0.6.0
     - julia
-    - numpy >=1.14.0
+    - {{ pin_compatible('numpy', min_pin='1.14') }}
```

The minimum cannot be lower than the numpy version used to build. For conda-forge, this is [currently](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/097cb77f27b76c910f7f7f50a42c67b8e54e787e/recipe/conda_build_config.yaml#L606-L610) numpy 1.21, as we follow NEP 29
```suggestion
    - {{ pin_compatible('numpy') }}
```

---

For the record, this would have worked, but it would have been useless and misleading.